Dockerfile

```diff
@@ -1,11 +1,20 @@
-FROM python:alpine3.15 AS base
+FROM python:3.8.5-slim-buster AS base
 MAINTAINER Gengo Dev Team
-RUN apk add build-base linux-headers pcre-dev
-RUN pip install uwsgi==2.0.18 && pip install awscli
+
+ARG uid=1000
+
+RUN apt-get -y update && apt-get -y install build-essential
+
+RUN pip install --upgrade pip
+
+RUN adduser -u ${uid} --disabled-password --disabled-login --gecos python python
+USER python
+
+RUN pip install --no-warn-script-location uwsgi==2.0.21 awscli
 
 FROM base as app
```

Comment:
neither `base` nor `app` is used
With this line, `apt-get`, `adduser`, etc. above are all meaningless.

Reply:
oh, maybe I'm wrong.
It's not meaningless, but there is no need to have 2 `FROM`s

Reply:
`uwsgi` and `awscli` are installed with `requirements.txt`.
This line should be deleted.
```
RUN pip install --no-warn-script-location uwsgi==2.0.21 awscli
```

Reply:
Adduser is needed for the Python Script Warning 
Apt-get update also need for build-essentials

Reply:
https://github.com/gengo/geoip-lite-api/pull/6 all of the change requested are here @ninotoshi